From bd8559a4722825b4a1fdb33042b90b63295ef835 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Caol=C3=A1n=20McNamara?= <caolanm@redhat.com>
Date: Thu, 17 Dec 2015 16:33:41 +0000
Subject: [PATCH 1/2] these popups should start invisible and take default
 popup menu bits

so they can position correctly under wayland

Change-Id: Idf65c0ef27be5aa6027a516207fd34c2e03407ca
---
 svx/uiconfig/ui/floatingundoredo.ui       | 1 -
 svx/uiconfig/ui/paralinespacingcontrol.ui | 2 --
 2 files changed, 3 deletions(-)

diff --git a/svx/uiconfig/ui/floatingundoredo.ui b/svx/uiconfig/ui/floatingundoredo.ui
index 66f30a6..2c48f82 100644
--- a/svx/uiconfig/ui/floatingundoredo.ui
+++ b/svx/uiconfig/ui/floatingundoredo.ui
@@ -11,7 +11,6 @@
     <property name="destroy_with_parent">True</property>
     <property name="type_hint">popup-menu</property>
     <property name="skip_pager_hint">True</property>
-    <property name="decorated">False</property>
     <property name="deletable">False</property>
     <child>
       <object class="GtkBox" id="box">
diff --git a/svx/uiconfig/ui/paralinespacingcontrol.ui b/svx/uiconfig/ui/paralinespacingcontrol.ui
index 5d7402b..67ca9c1 100644
--- a/svx/uiconfig/ui/paralinespacingcontrol.ui
+++ b/svx/uiconfig/ui/paralinespacingcontrol.ui
@@ -45,7 +45,6 @@
     <property name="icon-size">1</property>
   </object>
   <object class="GtkWindow" id="ParaLineSpacingControl">
-    <property name="visible">True</property>
     <property name="can_focus">False</property>
     <property name="hexpand">True</property>
     <property name="vexpand">True</property>
@@ -54,7 +53,6 @@
     <property name="destroy_with_parent">True</property>
     <property name="type_hint">popup-menu</property>
     <property name="skip_pager_hint">True</property>
-    <property name="decorated">False</property>
     <property name="deletable">False</property>
     <child>
       <object class="GtkGrid" id="grid2">
-- 
2.5.0

